pkg/cmd/providerenv/options.go

```diff
@@ -153,10 +162,10 @@ func (o *options) Run(f util.Factory) error {
 		return err
 	}
 
-	return execTmpl(o, shoot, secret, cloudProfile, messages)
+	return outputProviderEnv(o, shoot, secret, cloudProfile, messages)
 }
 
-func execTmpl(o *options, shoot *gardencorev1beta1.Shoot, secret *corev1.Secret, cloudProfile *gardencorev1beta1.CloudProfile, messages ac.AccessRestrictionMessages) error {
+func outputProviderEnv(o *options, shoot *gardencorev1beta1.Shoot, secret *corev1.Secret, cloudProfile *gardencorev1beta1.CloudProfile, messages ac.AccessRestrictionMessages) error {
```

Comment:
I would use `print` instead of `output` as verb for this function. The term output could be both the verb and the objects whch are printed. To me it more refers to the object(s) which are printed. Also in kubectl they more often use always `print` I think.
https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/get/get.go#L458-L559
https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/get/get.go#L260-L262

Reply:
✅